executor/executor_freebsd.cc

```diff
@@ -20,11 +20,23 @@
 #endif
 
 #include <signal.h>
+
```

Comment:
Please remove empty line here, we generally keep includes tight if there is no specific reason to add a new line.